test/protocol/ProtocolInitializationHandlerTest.js

```diff
@@ -328,4 +343,163 @@ describe("ProtocolInitializationHandler", async function () {
       });
     });
   });
+
+  describe("initV2_2_0", async function () {
+    let deployedProtocolInitializationFacet;
+    let configHandler;
+    let facetCut;
+
+    beforeEach(async function () {
+      version = "2.1.0";
+
+      // Deploy mock protocol initialization facet which simulates state before v2.2.0
+      const ProtocolInitilizationContractFactory = await ethers.getContractFactory("MockProtocolInitializationFacet");
+      const mockInitializationFacetDeployed = await ProtocolInitilizationContractFactory.deploy(
+        await getFees(maxPriorityFeePerGas)
+      );
+
+      await mockInitializationFacetDeployed.deployTransaction.wait();
+
+      const facetNames = [
+        "SellerHandlerFacet",
+        "AgentHandlerFacet",
+        "DisputeResolverHandlerFacet",
+        "OfferHandlerFacet",
+        "PauseHandlerFacet",
+        "FundsHandlerFacet",
+        "ExchangeHandlerFacet",
+      ];
+
+      const facetsToDeploy = await getFacetsWithArgs(facetNames);
+
+      // Make initial deployment (simulate v2.1.0)
+      await deployAndCutFacets(
+        protocolDiamond.address,
+        facetsToDeploy,
+        maxPriorityFeePerGas,
+        version,
+        mockInitializationFacetDeployed,
+        []
+      );
+
+      // Deploy v2.2.0 facets
+      const deployedFacets = await deployProtocolFacets(
+        ["ProtocolInitializationFacet", "ConfigHandlerFacet"],
+        { ProtocolInitializationFacet: [version, [], [], true] },
```

Comment:
```suggestion
        {},
```
initialize data for ProtocolInitializationFacet isn't handled on `deployProtocolFacets` (we check if facetName !== "ProtocolInitializationFacet")

Reply:
Changed.